flutter/pubspec.yaml

```diff
@@ -30,6 +30,7 @@ dependencies:
   collection: ^1.16.0
   web: ^1.1.0
   jni: 0.14.1
+  image_picker: ^1.1.2 # SentryFeedbackWidget
```

Comment:
what;s the effort of us doing this ourselves? I'd rather not introduce another dependency if possible

Reply:
Maybe [this](https://pub.dev/packages/vendor) can be a solution with a low overhead?

Reply:
Otherwise the effort is actually not insignificant. Can’t you leverage the native SDKs for this functionality?